src/protections/NsfwProtection.ts

```diff
@@ -65,8 +65,10 @@ export class NsfwProtection extends Protection {
             return;
         }
 
+        let shouldQuarantine = false;
```

Comment:
we should add information about the quarantine option to the description above (somewhere around line 42 I think?)